src/utils.js

```diff
@@ -116,6 +116,7 @@ export async function* withPage(percy, url, callback, retry) {
   let page = yield percy.browser.page({
     networkIdleTimeout: percy.config.discovery.networkIdleTimeout,
     requestHeaders: getAuthHeaders(percy.config.discovery),
+    captureServiceWorker: percy.config.captureServiceWorker,
```

Comment:
Note: this would mean we're using global value here, and in the CLI PR we're allowing `per-snapshot level config.

Reply:
So per snapshot level doesn't make sense for storybook SDK, since we'll need to create fresh new page. 

so we'll have to remove per-snapshot level support from CLI or we can keep it, but it will create confusion.